static/app/views/dashboards/widgetCard/chart.tsx

```diff
@@ -370,7 +370,10 @@ class WidgetCardChart extends Component<WidgetCardChartProps> {
     const bucketSize = getBucketSize(timeseriesResults);
 
     const valueFormatter = (value: number, seriesName?: string) => {
-      const aggregateName = seriesName?.split(':').pop()?.trim();
+      const decodedSeriesName = seriesName
+        ? WidgetLegendNameEncoderDecoder.decodeSeriesNameForLegend(seriesName)
+        : seriesName;
+      const aggregateName = decodedSeriesName?.split(':').pop()?.trim();
```

Comment:
Not entirely sure if this line is related to your change or not, but does the `':'` also have to get replaced with your delimiter here?

Also, are we using `,` in any special way? I saw something about a series list delimiter. If you group by multiple values those groupings (e.g. group by organization_slug and project) and you get a legend item like `org_slug,project` iirc.

Reply:
this `:` is not related to the change, that has to do with other legend names :) 

also `,` is fine because we're encoding all the series names before using that delimiter so it doesn't get messed up.